### PR TITLE
Make concept-published dual-stack  + helm integration

### PR DESCRIPTION
**Pull request by MMoisa**
- reworked the way concept-publisher accesses the transformers
- added vendoring
- updated gtg endpoint
- added build-info endpoint
- migrated to use helm packaging, see https://docs.google.com/document/d/1FrdjeQ9LGgrEWaRDkncBXJVMY_lXs56ngDShnM7Rqc8/edit#
- Jenkins job: https://k8s-jenkins-poc.in.ft.com/job/k8s-deployment/job/concept-publisher-auto-deploy/

### CODE REVIEW

**Review by tamas-molnar — APPROVED** [review]

**Review by kovacshuni — APPROVED** [review]

File: helm/concept-publisher/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        app: {{ .Values.service.name }}
+        visualize: "true" 
+    spec:
+      containers: 
+      - name: {{ .Chart.Name }} 
+        image: "{{ .Values.image.repository }}:{{ required "Please provide a valid docker image version by adding the argument at install '--set image.version=...'" .Values.image.version }}" 
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env: 
+        - name: PROXY_ADDRESS
+          value: "http://kafka:8082"
```
Comment (sbuliarca) [inline helm/concept-publisher/templates/deployment.yaml]: In the global-config ConfigMap we have the kafka.proxy.url.with.protocol key. Let's use that in here.

File: helm/concept-publisher/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        - name: TOPIC
```
Comment (sbuliarca) [inline helm/concept-publisher/templates/deployment.yaml]: I think we should make these configurable values.

**Review by kovacshuni — APPROVED** [review]

**Comment by kovacshuni** [comment]
Okay, too much diffs here. How about we create separate PRs to be from `helm` branch into `k8s-routing` to review easier.

File: README.md [hunk 3]
```diff
@@ -5,21 +5,21 @@ __Fetches concepts from concept-transformers and adds them to kafka.__
...
 ## Installation
 
-`go get github.com/Financial-Times/concept-publisher`
-
-## Running locally
-
 ```
-go build
+go get -u github.com/kardianos/govendor
+go get -u github.com/Financial-Times/concept-publisher
```
Comment (izzyblues) [inline README.md]: A very minor comment. This command is going to download the dependencies, which potentially can create mess in your `GOPATH` with vendoring. 
Probably it make sense to substitute this line with:
`git clone git@github.com:Financial-Times/concept-publisher.git`

File: vendor/vendor.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+			"path": "github.com/Financial-Times/service-status-go",
+			"revision": "3f5199736a3d7ae52394c63aac36834786825e21",
+			"revisionTime": "2016-03-23T11:15:42Z",
+			"tree": true,
+			"version": "0.1.0",
+			"versionExact": "0.1.0"
+		},
+		{
+			"checksumSHA1": "DYv6Q1+VfnUVxMwvk5IshAClLvw=",
+			"path": "github.com/Sirupsen/logrus",
```
Comment (izzyblues) [inline vendor/vendor.json]: It is interesting that it works uppercase

**Review by izzyblues — APPROVED** [review]
Looks good to me. I made only a minor comment about the readme.

**Pull request merged**